Assets/Scripts/GameManager.cs

```diff
@@ -91,8 +94,8 @@ private void LevelUpdate()
 
     private void NextLevel()
     {
-        _currentLevel += 1;
-        _UIManager.LevelText(_currentLevel);
+        CurrentLevel += 1;
+        _UIManager.LevelText(CurrentLevel);
```

Comment:
and in here you could actually do `_UIManager.LevelText(++CurrentLevel);` which would mean, increment CurrentLevel by 1, then pass it as input to LevelText